flocker/apiclient/test/test_client.py

```diff
@@ -403,16 +405,17 @@ def test_list_leases(self):
                 ])
             d.addCallback(lambda _: self.client.release_lease(d2))
             d.addCallback(lambda _: self.client.list_leases())
+            d.addCallback(frozenset)
             d.addCallback(
-                self.assertItemsEqual,
-                [
+                self.assertEqual,
```

Comment:
Never mind figured it out. I did a tiny repo and I found out that 1 field PClasses are orderable, whereas 2+ field PClasses are not orderable... which makes sense, but was more magic than I was expecting.